apps/src/lib/ui/CheckedRadioButton.jsx

```diff
@@ -8,7 +8,7 @@ export class CheckedRadioButton extends Component {
     disabledMode: PropTypes.bool,
     checked: PropTypes.bool.isRequired,
     value: PropTypes.string.isRequired,
-    onRadioButtonChange: PropTypes.func.isRequired
+    onRadioButtonChange: PropTypes.func
```

Comment:
Why did we need to change this and related properties to not be required?  Do we now need to handle when this is undefined (e.g. line 15 below)?

Reply:
There is a disabled mode where onChange is not necessary (for the student view), I was on the fence about the best way to handle this. Part of me thought that if it's undefined and getting called it'd be good if there were errors so I didn't add a check. Or maybe I could wrap the call in `if(!disabledMode)`, any thoughts?

Reply:
Ah, I see.  I don't have a strong opinion but it would be great to have this relationship between disabledMode and onRadioButtonChange noted somewhere, either in a comment or in code.